src/components/ReportingConfig/ReportingConfigForm.test.jsx

```diff
@@ -90,4 +90,19 @@ describe('<ReportingConfigForm />', () => {
     expect(wrapper.find('input#sftpFilePath').hasClass('is-invalid')).toBeTruthy();
     expect(wrapper.find('input#encryptedSftpPassword').hasClass('is-invalid')).toBeTruthy();
   });
+  it('Does not let you select a new value for data type if it uses the old progress_v1', () => {
```

Comment:
Minor: should we test that the new progress type can have its value changed?

Reply:
test added!